apps/passport-client/components/shared/cards/DevconnectTicket.tsx

```diff
@@ -0,0 +1,57 @@
+import { EdDSATicketPCD, getQRCodeColorOverride } from "@pcd/eddsa-ticket-pcd";
+import { QRDisplayWithRegenerateAndStorage } from "@pcd/passport-ui";
+import { useCallback } from "react";
+import styled from "styled-components";
+
+function makeEncodedVerifyLink(ticketId: string): string {
+  const link = `${window.location.origin}/checkin?id=${encodeURIComponent(
+    ticketId
+  )}`;
+  return link;
+}
+
+function TicketQR({ pcd }: { pcd: EdDSATicketPCD }) {
+  const generate = useCallback(async () => {
+    console.log(`[QR] generating proof, timestamp ${Date.now()}`);
+    const ticketId = pcd.claim.ticket.ticketId;
+    const verificationLink = makeEncodedVerifyLink(ticketId);
+    return verificationLink;
+  }, [pcd]);
+
+  return (
+    <QRDisplayWithRegenerateAndStorage
+      generateQRPayload={generate}
+      maxAgeMs={1000 * 60}
```

Comment:
at some point we should probably just not have these regenerate

Reply:
The regeneration was necessary when QR codes were generated from semaphore signatures with timestamps in the message. We don't have those for ticket PCDs, but perhaps we should have them (for the same reason that we should have them in feed credentials).